be/src/exec/pipeline/exchange/exchange_merge_sort_source_operator.cpp

```diff
@@ -15,7 +15,7 @@ Status ExchangeMergeSortSourceOperator::prepare(RuntimeState* state) {
     SourceOperator::prepare(state);
     _stream_recvr = state->exec_env()->stream_mgr()->create_recvr(
             state, _row_desc, state->fragment_instance_id(), _plan_node_id, _num_sender,
-            config::exchg_node_buffer_size_bytes, _runtime_profile, true, nullptr, true, true);
+            config::exchg_node_buffer_size_bytes, _runtime_profile, true, nullptr, true, -1, true);
```

Comment:
add some comment to explain why -1 is used here

Reply:
ok